monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -511,15 +520,20 @@ def generate(
                 gen_algo.set_data_stats(data_stats)
                 gen_algo.set_data_source(data_src_cfg)
                 name = f"{gen_algo.name}_{f_id}"
-                if gpu_customization:
-                    gen_algo.export_to_disk(
-                        output_folder,
-                        name,
-                        fold=f_id,
-                        gpu_customization=True,
-                        gpu_customization_specs=gpu_customization_specs,
-                    )
+                gen_algo.pre_check_skip_algo()
+                if not gen_algo.skip_bundlegen:
```

Comment:
Maybe we can commit less line changes by:
```
if gen_algo.skip_bundlegen:
	continue

if gpu_customization:
	...
```

Reply:
thanks, updated